projects/demo/src/assets/process-icons.js

```diff
@@ -61,11 +61,16 @@ function wrapIcon(source, name) {
 
     const width = parseInt(widthString, 10);
```

Comment:
There's also `process-icons.ts` which is used by `SvgService` when you define icon on the fly. Same changes need to be applied there.